server/templates/header.html

```diff
@@ -27,4 +29,15 @@
 <div class="container">
         <a href="/"><img src="/static/banner_small.png" class="img-fluid"><img></a>
 </div>
+
+
+
+<!-- Load React. -->
+<!-- Note: when deploying, replace "development.js" with "production.min.js". -->
```

Comment:
What does this comment mean? We need to replace a file when we want to `eb deploy`?